CHANGELOG.md

```diff
@@ -27,6 +27,7 @@
 
 - [CVE-2023-26136] Add resolution for tough-cookie to ^4.1.3 ([#889](https://github.com/opensearch-project/oui/pull/889))
 - [CVE-2023-26115] Bump word-wrap from 1.2.3 to 1.2.4 ([#891](https://github.com/opensearch-project/oui/pull/891))
+- Bump `node-sass` to a version based on `libsas@3.6.5` ([#912](https://github.com/opensearch-project/oui/pull/912))
```

Comment:
```suggestion
- Bump `node-sass` to a patched version based on `libsas@3.6.5` ([#912](https://github.com/opensearch-project/oui/pull/912)). See [patch commit](https://github.com/AMoo-Miki/node-sass/commit/43c74c0966b05c1e21a1e5e20a0c467ec8e669b4) for details.
```

Reply:
Also, should this reference specific CVEs resolved?

Reply:
libsas***s***?

Reply:
> Also, should this reference specific CVEs resolved?

There were too many of them IIRC.